### PR TITLE
Add Potential IC job expectation

### PR DESCRIPTION
**Pull request by xoza**

### CODE REVIEW

File: organization/roles/potential_individual_contributor_role.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Potential  IC
+
+## Job Purpose (Why HL needs your role)
+
+As a **high potential** software developer, we expect you to work closely with your mentor and team being greedy for receiving guidance, learning and growth. You will contribute through all the software development journey, focusing on work within your purpose and specialization team in improving your technical skills, taking ownership of your learning and tasks, optimizing and performing the standards and well-defined assignments and solving routine problems.
```
Comment (afontcu) [inline organization/roles/potential_individual_contributor_role.md:5]: I think "greedy" has some sort of negative connotation, so I suggest replacing it with "eager" (also tweak the sentence a bit 😇)

```suggestion
As a **high potential** software developer, we expect you to work closely with your mentor and team and to be eager to receive guidance, and learning and growth support. You will contribute through all the software development journey, focusing on work within your purpose and specialization team in improving your technical skills, taking ownership of your learning and tasks, optimizing and performing the standards and well-defined assignments and solving routine problems.
```

File: organization/roles/potential_individual_contributor_role.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  * **Work within your purpose and discipline team**
+    * Communicate well with all members. Proactively participate in collective working sessions, and communicate appropriately with all holaluz people. 
+    * Participate in the sprint tasks; doesn’t get caught up in the unknown, works to figure it out, asks questions without fear. 
+    * Perform spring well defined tasks,  help experienced people to solve more complex assignment or projects.
```
Comment (afontcu) [inline organization/roles/potential_individual_contributor_role.md:10]:
```suggestion
    * Perform well-defined sprint tasks, help experienced people to solve more complex assignment or projects.
```

What's the expectation of a Potential IC helping more experienced people? Should it read "collaborate with experienced people to solve more complex…"?

Comment (anxodio) [reply organization/roles/potential_individual_contributor_role.md:10]: I think so, help is strange in this context

Comment (afontcu) [reply organization/roles/potential_individual_contributor_role.md:10]: Then the suggestion would look as follows :)

```suggestion
    * Perform well-defined sprint tasks, collaborate with experienced people to solve more complex assignments or projects.
```

**Pull request merged**